sub-pages/submit-a-request/index.html

```diff
@@ -19,15 +19,15 @@
 
 {% block content_main %}
 
-    {% if initiative.title %}
-        <h1>{{ initiative.title | safe }}</h1>
+    {% if sub_page.title %}
+        <h1>{{ sub_page.title | safe }}</h1>
     {% endif %}
 
-    {% if initiative.content %}
+    {% if sub_page.content %}
     <section class="initiative_description block block__flush-top">
-        {{ initiative.content | safe }}
+        {{ sub_page.content | safe }}
 
-        {# TODO: Extra Content. Remove once content updates. #}
+        <!-- Extra Content. Remove once content updates. -->
```

Comment:
Nope. Dunno how that snuck back in. 

Although that whole section is going away once we move to an actual template. Content needs to stick those steps into their content section.